### PR TITLE
Update eppo js commons dependency to v1.4.1 to include allocation key in logged event's experiment attribute (FF-858) 

### PR DESCRIPTION
**Pull request by sameerank**
---
labels: mergeable
---
[//]: #  (Link to the issue corresponding to this chunk of work)
Fixes: [FF-858](https://linear.app/eppo/issue/FF-858/update-javascript-sdk-with-allocation-key)

## Motivation and Context
[//]: #  (Why is this change required? What problem does it solve?)

## Description
[//]: # (Describe your changes in detail)

## How has this been tested?
[//]: # (Please describe in detail how you tested your changes)

[//]: # (OPTIONAL)
[//]: # (Add one or multiple labels: enhancement, refactoring, bugfix)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "xhr-mock": "^2.5.1"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "^1.3.1",
+    "@eppo/js-client-sdk-common": "^1.4.1",
```
Comment (sameerank) [inline package.json:59]: I might bump this a higher version if this gets merged/published first: https://github.com/Eppo-exp/js-client-sdk-common/pull/13

File: test/testHelpers.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import * as fs from 'fs';
 
-import { IExperimentConfiguration } from '../src/dto/experiment-configuration-dto';
```
Comment (sameerank) [inline test/testHelpers.ts:3]: These imports were broken

**Review by leoromanovsky — APPROVED** [review]
very nice!

File: Makefile [hunk 3]
```diff
@@ -25,12 +25,18 @@ help: Makefile
...
+githubRepoLink := https://github.com/Eppo-exp/sdk-test-data.git
 .PHONY: test-data
-test-data:
+test-data: 
 	rm -rf $(testDataDir)
-	mkdir -p $(testDataDir)
-	gsutil cp gs://sdk-test-data/rac-experiments-v2.json $(testDataDir)
-	gsutil cp -r gs://sdk-test-data/assignment-v2 $(testDataDir)
+	mkdir -p $(tempDir)
+	git clone -b ${branchName} --depth 1 --single-branch ${githubRepoLink} ${gitDataDir}
```
Comment (aarsilv) [inline Makefile:36]: 🔥

File: test/testHelpers.ts (hunk 2 above)
Comment (aarsilv) [reply test/testHelpers.ts:3]: 😞

File: test/testHelpers.ts [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-import { IVariation } from '../src/experiment/variation-dto';
+import { IExperimentConfiguration } from '@eppo/js-client-sdk-common/dist/dto/experiment-configuration-dto';
+import { IVariation } from '@eppo/js-client-sdk-common/dist/dto/variation-dto';
 
 export const TEST_DATA_DIR = './test/data/';
 export const ASSIGNMENT_TEST_DATA_DIR = TEST_DATA_DIR + 'assignment-v2/';
-export const MOCK_RAC_RESPONSE_FILE = 'rac-experiments-v2.json';
+export const MOCK_RAC_RESPONSE_FILE = 'rac-experiments-v3.json';
```
Comment (aarsilv) [inline test/testHelpers.ts:8]: 📈

**Review by aarsilv — APPROVED** [review]
PR title a tad misleading; really this PR just uses updated common js package so that we can include allocation key in logged event's experiment attribute. One more level of direction!

**Review by ploomis14 — APPROVED** [review]

**Pull request merged**